tests/input/html/test_html.py

```diff
@@ -0,0 +1,90 @@
+    image_file_path = IMAGE_FILE_PATH
+    assert manager.has_jpeg_preview(file_path=image_file_path) is True
+    path_to_file = manager.get_jpeg_preview(file_path=image_file_path, force=True)
+    assert os.path.exists(path_to_file) is True
+    assert os.path.getsize(path_to_file) > 0
+
+
+def test_to_pdf() -> None:
+    manager = PreviewManager(cache_folder_path=CACHE_DIR, create_folder=True)
+    image_file_path = IMAGE_FILE_PATH
+    assert manager.has_jpeg_preview(file_path=image_file_path) is True
+    path_to_file = manager.get_pdf_preview(file_path=image_file_path, force=True)
+    assert os.path.exists(path_to_file) is True
+
+
+def test_page_number() -> None:
+    manager = PreviewManager(cache_folder_path=CACHE_DIR, create_folder=True)
+    page_number = manager.get_page_nb(file_path=IMAGE_FILE_PATH)
+    assert page_number == 7
+
+
+def test_to_pdf_no_extension() -> None:
+    manager = PreviewManager(cache_folder_path=CACHE_DIR, create_folder=True)
+    image_file_path = IMAGE_FILE_PATH_NO_EXTENSION
+    assert manager.has_jpeg_preview(file_path=image_file_path) is True
+    path_to_file = manager.get_pdf_preview(file_path=image_file_path, force=True)
+    assert os.path.exists(path_to_file) is True
+
+
+def test_to_pdf_no_extension_extension_forced() -> None:
+    manager = PreviewManager(cache_folder_path=CACHE_DIR, create_folder=True)
+    image_file_path = IMAGE_FILE_PATH_NO_EXTENSION
+    assert manager.has_pdf_preview(file_path=image_file_path, file_ext=".txt") is True
+    path_to_file = manager.get_pdf_preview(file_path=image_file_path, force=True, file_ext=".txt")
+    assert os.path.exists(path_to_file) is True
+
+
+def test_to_jpeg_no_extension() -> None:
+    manager = PreviewManager(cache_folder_path=CACHE_DIR, create_folder=True)
+    image_file_path = IMAGE_FILE_PATH_NO_EXTENSION
+    assert manager.has_jpeg_preview(file_path=image_file_path) is True
+    path_to_file = manager.get_jpeg_preview(file_path=image_file_path, force=True)
+    assert os.path.exists(path_to_file) is True
+
+
+def test_to_jpeg_no_extension_extension_forced() -> None:
+    manager = PreviewManager(cache_folder_path=CACHE_DIR, create_folder=True)
+    image_file_path = IMAGE_FILE_PATH_NO_EXTENSION
+    assert manager.has_jpeg_preview(file_path=image_file_path, file_ext=".txt") is True
+    path_to_file = manager.get_jpeg_preview(file_path=image_file_path, force=True, file_ext=".txt")
+    assert os.path.exists(path_to_file) is True
+
+
+def test_page_number__no_extension() -> None:
+    manager = PreviewManager(cache_folder_path=CACHE_DIR, create_folder=True)
+    page_number = manager.get_page_nb(file_path=IMAGE_FILE_PATH_NO_EXTENSION)
+    assert page_number == 7
+
+
+def test_page_number__extension_forced() -> None:
+    manager = PreviewManager(cache_folder_path=CACHE_DIR, create_folder=True)
+    page_number = manager.get_page_nb(file_path=IMAGE_FILE_PATH_NO_EXTENSION, file_ext=".txt")
+    assert page_number == 7
```

Comment:
```suggestion

def test_to_pdf_no_extension_extension_forced() -> None:
    manager = PreviewManager(cache_folder_path=CACHE_DIR, create_folder=True)
    image_file_path = IMAGE_FILE_PATH_NO_EXTENSION
    assert manager.has_pdf_preview(file_path=image_file_path, file_ext=".html") is True
    path_to_file = manager.get_pdf_preview(file_path=image_file_path, force=True, file_ext=".html")
    assert os.path.exists(path_to_file) is True


def test_to_jpeg_no_extension() -> None:
    manager = PreviewManager(cache_folder_path=CACHE_DIR, create_folder=True)
    image_file_path = IMAGE_FILE_PATH_NO_EXTENSION
    assert manager.has_jpeg_preview(file_path=image_file_path) is True
    path_to_file = manager.get_jpeg_preview(file_path=image_file_path, force=True)
    assert os.path.exists(path_to_file) is True


def test_to_jpeg_no_extension_extension_forced() -> None:
    manager = PreviewManager(cache_folder_path=CACHE_DIR, create_folder=True)
    image_file_path = IMAGE_FILE_PATH_NO_EXTENSION
    assert manager.has_jpeg_preview(file_path=image_file_path, file_ext=".html") is True
    path_to_file = manager.get_jpeg_preview(file_path=image_file_path, force=True, file_ext=".html")
    assert os.path.exists(path_to_file) is True


def test_page_number__no_extension() -> None:
    manager = PreviewManager(cache_folder_path=CACHE_DIR, create_folder=True)
    page_number = manager.get_page_nb(file_path=IMAGE_FILE_PATH_NO_EXTENSION)
    assert page_number == 7


def test_page_number__extension_forced() -> None:
    manager = PreviewManager(cache_folder_path=CACHE_DIR, create_folder=True)
    page_number = manager.get_page_nb(file_path=IMAGE_FILE_PATH_NO_EXTENSION, file_ext=".html")
    assert page_number == 7
```
As you test an html file, extension force test should use .html extension. If not, preview may be runned with wrong builder or have unwanted behavior (guessed mimetype is wrong, so infilter doesn't apply for example).